tests/e2e/createWallets.spec.ts

```diff
@@ -76,16 +70,28 @@ test.describe("Create or connect wallets", () => {
     const createWalletButton = await getByText($document, "Continue");
     createWalletButton.click();
 
-    await welcomePage.waitForResponse(() => true);
+    await page.waitForResponse(() => true);
 
     await waitFor(() => getByText($document, "Your Alby account is ready."));
 
-    await commonCreateWalletSuccessCheck({ browser, welcomePage, $document });
+    await commonCreateWalletSuccessCheck({ page, $document });
+
+    // reload and expect to be on publishers screen
+    await Promise.all([
```

Comment:
Why do we add this in this test? 
can this be a new test

Reply:
Yeah, you're right. But in reality this won't work without adding a wallet first.  
We could fake it, split this into two tests and make it dependent on each other:
https://playwright.dev/docs/api/class-test#test-describe-serial

Reply:
hmm.. but won't we need for anything an account first? (so anything would go into this? ;))

Reply:
should be, no? in the end it's just an entry in the browser storage (setting some state)

Reply:
Doesn't seem to be too simple to use i.e. `browser.storage...` within puppeteer because in our the code it's added via the webextension-polyfill. I believe that's what we would need, right?  
[This is easier with i.e. localstoarge](https://stackoverflow.com/a/56141229/1667461).

Reply:
yeah, I see. this is the same problem as in the other e2e test for the batteries. 
hmmmmmmmmmmmm....